nipype/interfaces/fsl/epi.py

```diff
@@ -199,32 +199,28 @@ class TOPUPInputSpec(FSLCommandInputSpec):
     # TODO: the following traits admit values separated by commas, one value
     # per registration level inside topup.
     warp_res = traits.Float(
-        10.0, usedefault=True,
         argstr='--warpres=%f',
         desc=('(approximate) resolution (in mm) of warp '
               'basis for the different sub-sampling levels'
-              '.'))
-    subsamp = traits.Int(1, usedefault=True,
-                         argstr='--subsamp=%d', desc='sub-sampling scheme')
+              '(default in FSL 10).'))
```

Comment:
@effigies - yes this is probably the main problem, some flags shouldn't be used at all as default. I will review this later and add the table you suggested. It is extra work for us to keep this updated, but it might be the only solution that make sense...

This is not the only interface that might have similar problems (at least not the only one that possibly was changed after my PR with `usedefault=True`)

Reply:
Maybe put a big warning that the values may be out-of-date? We're not going to be able to regularly update it, but letting people know there's an issue worth looking into and what we knew at this point seems like the best we can do.